src/test/java/org/folio/innreach/controller/InnReachTransactionControllerTest.java

```diff
@@ -2073,4 +2137,35 @@ private static Stream<Arguments> checkOutItemTestArguments() {
     );
   }
 
+  private ResultList<RequestDTO> getOpenRequests() {
+    ResultList<RequestDTO> resultList = new ResultList<>();
+    RequestDTO request = new RequestDTO();
+    request.setStatus(OPEN_NOT_YET_FILLED);
+    request.setRequestType(RequestDTO.RequestType.RECALL.getName());
+    List<RequestDTO> list = new ArrayList<>();
+    list.add(request);
+    resultList.setResult(list);
+    return resultList;
```

Comment:
it can be simplified to

```
  private ResultList<RequestDTO> getOpenRequests() {
    var request = new RequestDTO();
    request.setStatus(OPEN_NOT_YET_FILLED);
    request.setRequestType(RequestDTO.RequestType.RECALL.getName());
    return ResultList.asSinglePage(request);
  }
```

Reply:
Done